src/scripts/csvToTxtJson.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import fs from 'fs';
+import { pipeline } from 'stream';
+
+import { reCreateFolder } from '../utils/reCreateFolder.js';
+
+const CSV_SOURCE = `${process.cwd()}/assets/books.csv`;
+const TXT_TARGET_FOLDER = `${process.cwd()}/output/`;
```

Comment:
as an alternative, path.resolve() can be used

Reply:
added this and the previous suggestion to #2 PR.